src/app/auth/login.ts

```diff
@@ -228,3 +383,16 @@ export const isCodeValid = async ({
 
   return TwilioClient().validateVerify({ to: phone, code })
 }
+
+export const verifyJwt = async (token: string) => {
+  const newToken = await sendOathkeeperRequest(token as SessionToken)
+  if (newToken instanceof Error) return newToken
+  const keyJwks = await jwksRsa(getJwksArgs()).getSigningKey()
+  const verifiedToken = jsonwebtoken.verify(newToken, keyJwks.getPublicKey(), {
+    algorithms: ["RS256"],
+  })
+  if (typeof verifiedToken === "string") {
+    throw new JwtVerifyTokenError("tokenPayload should be an object")
```

Comment:
why are we throwing in app layer? please use a proper return and include return type